README.md

```diff
@@ -74,4 +79,8 @@ In order to generate the doxygen documentation please follow the next steps:
 on the project build directory run:
 
 cmake -DBUILD_DOCUMENTATION=ON ..
-make doc
\ No newline at end of file
+make doc
+
+## Versioning and Releases
+From now on, this library uses Semantic Versioning as described at [semver.org](https://semver.org).
```

Comment:
From now on sounds weird in a seemingly timeless README. Just drop it.

Reply:
Doesn't seem to be done, still says "from now on".

Reply:
ah, sorry i've misread the comment. Should be done now